### PR TITLE
Add biscuit puzzle

### PR DESCRIPTION
**Pull request by SyntappZ**

### CODE REVIEW

File: puzzles/puzzle1.html [hunk 1]
```diff
@@ -30,113 +30,145 @@
...
-    <img src='https://avatars2.githubusercontent.com/u/47202564?s=460&u=35c8f87e6b9e00267c22664efd668db3b4ad7491&v=4' alt='davidProfPic'>
-    <p>"Alone we can do so little; together we can do so much." – Helen Keller</p>
-  </div>
-
-  <div class='camperCell' id='camper2'>
-    <h1>Camper2</h1>
-    <img src='https://avatars1.githubusercontent.com/u/583231?s=460&u=a59fef2a493e2b67dd13754231daf220c82ba84d&v=4' alt='davidProfPic'>
-    <p>“Be the change that you wish to see in the world.” ― Mahatma Gandhi</p>
-  </div>
+      <div class="camperCell" id="camper2">
```
Comment (DavidMatthewFraser) [inline puzzles/puzzle1.html:164]: Nice! Thanks for doing the puzzle! 
One thing: looks like you're using the same 'id' as camper 2. Try adding your own id and custom style - which will be a walk in the park for you 😄.

Comment (SyntappZ) [reply puzzles/puzzle1.html]: sorted that now mate 😎👍

File: index.html [hunk 2]
```diff
@@ -46,12 +46,12 @@ <h3>Easy to HTML</h3>
...
             </form>
           </div>
-      
+          
           
         <h2 class="greeting">Hello from Nowhereland :-)</h2>
         <p>Ha! for future writers, remember that the javascript on the head shouldn't include things like event handlers or
           the code can potentially break!</p>
-            <a href='https://davidmatthewfraser.github.io/easy-to-git/puzzles/puzzle1'><button class='btn btn-deault'>Puzzle One Prototype</button></a>
+            <a href='puzzles/puzzle1.html'><button class='btn btn-deault'>Puzzle One Prototype</button></a>
```
Comment (DavidMatthewFraser) [inline index.html:54]: This is better, thanks!

File: puzzles/puzzle1.html [hunk 3]
```diff
@@ -30,113 +30,150 @@
...
+    #davidfraser {
       color: blue;
     }
 
-    #camper2{
+    #camper2 {
       color: purple;
     }
-
+    #biscuit {
```
Comment (DavidMatthewFraser) [inline puzzles/puzzle1.html:96]: Cool Style!

**Comment by DavidMatthewFraser** [comment]
Good Job!

**Pull request merged**